torch/jit/annotations.py

```diff
@@ -1,6 +1,7 @@
 import ast
 import enum
 import inspect
+import logging
```

Comment:
```suggestion
import warnings
```

Reply:
Done. 

I think most of the times (by default) logging.warning and warning.warn behave exactly the same, unless configured otherwise:

https://stackoverflow.com/a/37979724

And not prompting user to set the environment variable is on purpose, the feature isn't complete after all.